site/content/docs/snippets/_index.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Snippets Overview
+weight: 70
+slug: /
+aliases:
+- /docs/snippets
+- /docs/browser-checks/partials-code-snippets/
+
+menu:
+  docs:
+    parent: "Snippets"
+---
+
+Snippets are Javascript files which can be reused across checks. Snippets can be used in [browser checks](/docs/browser-checks/), as well as [setup and teardown scripts](/docs/api-checks/setup-teardown-scripts/) for API checks. By reusing code with snippets, checks can be made more maintainable and easier to develop.
+
+Snippets can be particularly useful for:
+
+- a common login procedure
+- a common navigation flow
+- a common setup or teardown procedure
+
+### How to use snippets
+
+To create a snippet, open the <a href="https://app.checklyhq.com/snippets" target="_blank">snippets page</a>. This can be accessed from the toolbar on the left side of the UI. When creating a snippet, note that the snippet name will be used as its filename. It's not necessary to name the snippet with the `.js` file extension, though, since this will be added automatically.
```

Comment:
```suggestion
To create a snippet, access <a href="https://app.checklyhq.com/snippets" target="_blank">the snippets section on the left side of the UI</a>. 

When creating a snippet, note that the snippet name will be used as its filename. It's not necessary to name the snippet with the `.js` file extension, though, since this will be added automatically.
```